documentation/assemblies/assembly-kafka-connect-authorization.adoc

```diff
@@ -0,0 +1,34 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+// assembly-deployment-configuration-kafka-connect-s2i.adoc
+
+[id='assembly-kafka-connect-authorization-{context}']
+
+= Kafka Connect user authorization
+
+If authorization is enabled for Kafka Connect, the `KafkaUser` resource for the Kafka Connect user must be configured to provide read/write access rights to the consumer group and
```

Comment:
Since this will not be always done in the KafkaUser, I would leave it out in this sentence.

```suggestion
If authorization is enabled for Kafka Connect, the Kafka Connect user must be configured to provide read/write access rights to the consumer group and
```